osmaxx/core/templates/pages/about_us.html

```diff
@@ -0,0 +1,28 @@
+{% extends "osmaxx/base.html" %}
+{% load i18n %}
+
+{% block main %}
+<div class="container-row content">
+    <h2>About OSMaxx</h2>
+    <p>
+        OSMaxx (short for "OpenStreetMap arbitrary excerpt export") cuts out OpenStreetMap data, processes it to
+        geodata, converts it to typical GIS fileformats and offers those for download.
+    </p>
+    <h3>Useful links</h3>
+    <p>Can be found in our separate <a href="{% url 'pages:downloads' %}">downloads page</a></p>
+
+    <h3>Impressum</h3>
```

Comment:
"Impressum" is German. English term would be "imprint". Though I'm not sure whether that would only be used in actually _printed_ material.